.ci/component_descriptor

```diff
@@ -11,6 +11,8 @@ echo "enriching creating component descriptor from ${BASE_DEFINITION_PATH}"
 eval "$(jq -r ".images |
   map(if (.name == \"hyperkube\") then
     \"--generic-dependencies '{\\\"name\\\": \\\"\" + .name + \"\\\", \\\"version\\\": \\\"\" + .tag + \"\\\"}'\"
+  elif (.repository | startswith(\"eu.gcr.io/gardener-project/gardener/external-admission-controller\")) then
+  \"--container-image-dependencies '{\\\"name\\\": \\\"\" + .name + \"\\\", \\\"image_reference\\\": \\\"\" + .repository + \":\" + .tag + \"\\\", \\\"version\\\": \\\"\" + .tag + \"\\\"}'\"
```

Comment:
on the side note - this is absolutely horrendous.

Reply:
Agreed 👍